### PR TITLE
Rename Nanotrasen uniform

### PR DESCRIPTION
**Pull request by Firefox13**
## About The Pull Request
Renames them and loadout entries

## Why It's Good For The Game
These items promoted roleplay that clashes with our existing setting.

## Testing
изображение
изображение

## Changelog
:cl:
del: nanotrasen security uniform
add: securitech security uniform (uses nanotrasen security uniform sprites)
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
Society

**Review by TheShown911 — APPROVED** [review]
Skill issue

File: html/changelog.html [hunk 1]
```diff
@@ -46,926 +46,6 @@
 		</td>
 	</tr>
 </table>
-
-			<h2 class="date">10 October 2022</h2>
```
Comment (ValoTheValo) [inline html/changelog.html:50]: Did you mean to delete like 900 lines of changelog?

File: html/changelog.html [hunk 2]
```diff
@@ -46,926 +46,6 @@
...
-
-			<h2 class="date">12 August 2022</h2>
-			<h3 class="author">Anonmare updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="tweak">The C20 is now in line with the standardised naming convention. Moebius gun products are now abbrviated to "ML"</li>
-			</ul>
-			<h3 class="author">null-Aurelian updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="rscadd">Add recipe for Trash Bag item</li>
-			</ul>
```
Comment (ValoTheValo) [inline html/changelog.html:968]: yeah

File: html/changelog.html (hunk 1 above)
Comment (Firefox13) [reply html/changelog.html:50]: lol, I did not, it's the bot

**Comment by Firefox13** [comment]
Okay, I fixed the merge conflict. Changelog bot acted a little weird...

**Review by Pink-Chink — APPROVED** [review]

**Comment by Firefox13** [comment]
It did it again anyway. I'm going to revert the commit and just merge the PR.
If it breaks anything I'll revert and re-do it.

**Pull request merged**